src/articles/dto/articles-response.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { Expose, Transform, TransformFnParams } from 'class-transformer';
+import { capitalizeTitle } from '../../utilities/capitalize-title';
+
+function shortenText({ value: text }: TransformFnParams) {
+  if (!text) {
+    return null;
+  }
+  if (text.length > 100) {
+    return `${text.substring(0, 98)}${'.'.repeat(3)}`;
```

Comment:
You can use the `U+2026` character instead of `'.'.repeat(3)`
https://www.compart.com/en/unicode/U+2026

But even if you want to use three dots, `'.'.repeat(3)` is over-engineering a simple problem.
This would be a lot better:
```
return `${text.substring(0, 98)}...`
```